lib/uksched/include/uk/sched.h

```diff
@@ -107,9 +107,9 @@ static inline void uk_sched_yield(void)
 	s->yield(s);
 }
 
-int uk_sched_thread_add(struct uk_sched *s, struct uk_thread *t);
+int uk_sched_thread_add(struct uk_sched *s, struct uk_thread *t) __nonnull;
 
-int uk_sched_thread_remove(struct uk_thread *t);
+int uk_sched_thread_remove(struct uk_thread *t) __nonnull;
```

Comment:
The function already uses `UK_ASSERT`, using `__nonnull` is redundant here and would generate avoidable warnings